wire-ios-transport/Source/Requests/ZMTransportRequest.m

```diff
@@ -853,7 +853,6 @@ - (void)setDebugInformationState:(NSObject *)state;
 - (void)addContentDebugInformation:(NSString *)debugInformation
 {
     [self.debugInformation addObject:debugInformation];
-    self.contentDebugInformationHash ^= debugInformation.hash;
 }
```

Comment:
This was a side effect, that needs to be done carefully. Therefore I mimic the old behaviour in the places calling `addContentDebugInformation `, see 36100be721cb014522b3eaf31f1bb31cac751ccd.

Reply:
what does `^=` do?

Reply:
It's a XOR operator